sym_api_client_python/clients/datafeed_client_v2.py

```diff
@@ -0,0 +1,57 @@
+from .api_client import APIClient
+import logging
+import json
+
+class DataFeedClientV2(APIClient):
+    def __init__(self, bot_client):
+        self.bot_client = bot_client
+        self.ackid = ""
+
+    def create_datafeed(self):
+        url = '/agent/v5/datafeeds'
+        headers = {'content-type': 'application/json'}
+        response = self.bot_client.execute_rest_call("POST", url, headers=headers)
+
+        datafeed_id = response.get("id")
+        logging.debug('DataFeedClient/create_datafeed() --> {}'.format(datafeed_id))
+        return datafeed_id
+
+
+    def read_datafeed(self, datafeed_id):
+        """
+        DF 2 Version: We need to use an ack Id to make sure the events are well received
+        """
+        logging.debug('DataFeedClient/read_datafeed()')
+        logging.debug('Before call Ack id {}'.format(self.ackid))
+        url = '/agent/v5/datafeeds/{0}/read'.format(datafeed_id)
+
+        headers = {'content-type': 'application/json'}
+        data = {}
+        if self.ackid == "":
+            data["ackId"] = ""
+        else:
+            data["ackId"] = self.ackid
+
+        datafeed_read = self.bot_client.execute_rest_call("POST", url, headers=headers, json=data)
+        #TODO: Rename the datafeed read to events or event_list as it is much more readable
```

Comment:
Please remove the TODO comments. IMHO it is quite confusing: either is has to be done and you do it or leave it as is